integration-tests/src/main/java/org/apache/polaris/service/it/env/RestApi.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.service.it.env;
+
+import jakarta.ws.rs.client.Invocation;
+import jakarta.ws.rs.client.WebTarget;
+import java.net.URI;
+import java.util.Map;
+
+public class RestApi {
```

Comment:
Should this class just be folded into PolarisApiClient? It seems like they're inherently coupled.

Reply:
Some coupling does exist, but it is not intentional. I'd prefer to refactor `PolarisApiClient` to be a simple container of URL / realm strings (and rename to `PolarisEndpoint`). WDYT?

Reply:
Hmm I think it would be simpler to just use OpenAPI generated clients as they already know the endpoints. We might already have this? https://github.com/apache/polaris/tree/main/api

Reply:
These test API helpers also perform some validation. I'm not sure that a generated client would be "simpler". In any case, let's defer this part until we settle on the basic test framework.